### ISSUE TITLE
ArticlePage Templates

### ISSUE THREAD

**Issue by Zsvoboda87** [coderedcorp/coderedcms#572]
#### Is your feature request related to a problem? Please describe.
If you're using an article page and want to change to different template the content is no longer wrapped in a container causing all the content to fill the whole page without the margins.

#### Describe the solution you'd like
Possibly more templates, or remove those options on that page model.  

#### Describe alternatives you've considered
I added an if statement on the webpage template checking the page type and conditionally added a container if neccessary

#### Additional context
This issue came when a client wanted to use a different template on their article page.

**Comment by vsalvino** [comment]
Was this an issue using one of the built-in CRX templates? The base template does not use containers for content, allowing the page templates to implement their own structures.

**Comment by rosslaird** [comment]
Yes, this occurs with the built-in article template. If a user chooses anything other than Default in the template dropdown for Visual Design, the page loses the container that is provided by the default article page layout. Consequently, options other than Default are somewhat unusable in the context of article pages; they are intended for use by the other page types. So, I think the idea (since this issue arose in the context of my site) is either to remove the layout options for article pages (since they don't work well anyway) or replace the currently non-functional layout options with options that provide greater functionality for those pages.

**Comment by vsalvino** [comment]
The use of that dropdown for templates is primarily for the developer to create custom alternate layouts, which can be hooked in to specific page types. However out of the box, we only provide a couple bare-bones options.

So yes the idea of having more templates for an Article is possible. However I don't anticipate we will provide any more out-of-the-box. If you'd like to develop your own, or change what is in that dropdown, you can follow this guide: https://github.com/coderedcorp/coderedcms/issues/44#issuecomment-524497132
